docs/gui/screens.md

```diff
@@ -20,33 +20,108 @@ Any GUI rendered by Minecraft is typically done using `GuiGraphics`. `GuiGraphic
+
 :::note
 Strings should typically be passed in as [`Component`s][component] as they handle a variety of usecases, including the two other overloads of the method.
 :::
 
 ### Textures
 
-Textures are drawn through blitting, hence the method name `#blit`, which, for this purpose, copies the bits of an image and draws them directly to the screen. These are drawn through a position texture shader. While there are many different `#blit` overloads, we will only discuss two static `#blit`s.
+Textures are drawn through blitting, hence the method name `#blit`, which, for this purpose, copies the bits of an image and draws them directly to the screen. These are drawn through a position texture shader.
+
+Each `#blit` method takes in a `ResourceLocation`, which represents the absolute location of the texture:
+
+```java
+// Points to 'assets/examplemod/textures/gui/container/example_container.png'
+private static final ResourceLocation TEXTURE = new ResourceLocation("examplemod", "textures/gui/container/example_container.png");
+```
+
+While there are many different `#blit` overloads, we will only discuss two of them.
 
-The first static `#blit` takes in six integers and assumes the texture being rendered is on a 256 x 256 PNG file. It takes in the left x and top y screen coordinate, the left x and top y coordinate within the PNG, and the width and height of the image to render.
+The first `#blit` takes in six integers and assumes the texture being rendered is on a 256 x 256 PNG file. It takes in the left x and top y screen coordinate, the left x and top y coordinate within the PNG, and the width and height of the image to render.
 
 :::tip
 The size of the PNG file must be specified so that the coordinates can be normalized to obtain the associated UV values.
 :::
 
-The static `#blit` which the first calls expands this to nine integers, only assuming the image is on a PNG file. It takes in the left x and top y screen coordinate, the z coordinate (referred to as the blit offset), the left x and top y coordinate within the PNG, the width and height of the image to render, and the width and height of the PNG file.
+The second `#blit` which the first calls expands this to seven integers and two floats for the PNG coordinates, only assuming the image is on a PNG file. It takes in the left x and top y screen coordinate, the z coordinate (referred to as the blit offset), the left x and top y coordinate within the PNG, the width and height of the image to render, and the width and height of the PNG file.
+
+#### `blitSprite`
+
+`#blitSprite` is a special implementation of `#blit` where the texture is written to the GUI texture atlas. Almost all textures that overlaid the background texture are now sprites. For comparison, the 'lit progress' and 'burn progress' textures used within furnaces are now sprites. All sprite textures are relative to `textures/gui/sprites` and do not need to specify the file extension.
```

Comment:
"are now" is bad, as "now" is an unclear term. Use something like "Most textures that are overlaying the background in some way, for example the burn progress overlay in furnace GUIs, are sprites."